smart-send-logistics/smart-send-blocks-integration.php

```diff
@@ -0,0 +1,233 @@
+<?php
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface;
+
+/**
+ * Class for integrating with WooCommerce Blocks
+ */
+    class smart_send_Blocks_Integration implements IntegrationInterface {
+
+	/**
+	 * The name of the integration.
+	 *
+	 * @return string
+	 */
+	public function get_name() {
+		return 'smart-send-logistics';
+	}
```

Comment:
Why do we need this function? This is hardcoded many other places.

Reply:
The get_name function is designed to return the name of the integration, specifically 'smart-send-logistics'. This name is crucial because it is used to identify our extension within the checkout endpoint of WooCommerce.

I have updated it and extracted it into a constant so in future if change the name we can do it from one place.